BuildAndTest.cmd

```diff
@@ -64,36 +64,37 @@ if "%ERRORLEVEL%" NEQ "0" (
 goto ExitFailed
 )
 
-msbuild /verbosity:minimal /target:rebuild src\Everything.sln /filelogger /fileloggerparameters:Verbosity=detailed /p:"RunBinSkim=false" /p:"BinSkimVerboseOutput=true"
-set Platform=AnyCPU
+msbuild /verbosity:minimal /target:BuildAndInjectObjectModel src\Sarif\Sarif.csproj /fileloggerparameters:Verbosity=detailed
+if "%ERRORLEVEL%" NEQ "0" (
+goto ExitFailed
+)
 
+msbuild /verbosity:minimal /target:rebuild src\Everything.sln /filelogger /fileloggerparameters:Verbosity=detailed /p:"RunBinSkim=false" /p:"BinSkimVerboseOutput=true"
 if "%ERRORLEVEL%" NEQ "0" (
 goto ExitFailed
 )
 
+set Platform=AnyCPU
```

Comment:
Minor, unrelated change: check the result of the `msbuild` command immediately after the command finishes. Don't set the environment variable until after that.